docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Migrating ASP.NET MVC Applications to Windows Containers
+description: Learn how to take an existing ASP.NET MVC application and run it in a Windows Docker Container
+keywords: .NET, .NET Server, .NET Desktop
```

Comment:
I'd think we need more powerful keywords here like Windows Containers, Docker, ASP.NET MVC